app/views/shelters/show.html.erb

```diff
@@ -7,3 +7,9 @@
 </ul>
 
 <a href="/shelters/<%= @shelter.id %>/edit">Update Shelter</a>
+
+<form action="/shelters/<%= @shelter.id %>" method="POST">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="hidden" name="_method" value="DELETE">
+  <input type="submit" value="Delete Shelter"/>
+</form>
```

Comment:
Right now edit is a link and delete is a button, which doesn't look nice. Need to make this uniform.